codalab/worker_manager/azure_batch_worker_manager.py

```diff
@@ -11,6 +11,7 @@
         TaskCounts,
         TaskContainerSettings,
     )
+    from azure.batch.models._models_py3 import BatchErrorException  # type: ignore
```

Comment:
I think this should just work:

```suggestion
    from azure.batch.models import BatchErrorException  # type: ignore
```

See https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/batch/azure-batch/azure/batch/models/__init__.py#L27